com.notepadqq.Notepadqq.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.notepadqq.Notepadqq
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: notepadqq
+rename-icon: notepadqq
+rename-desktop-file: notepadqq.desktop
+rename-appdata-file: notepadqq.appdata.xml
+copy-icon: true
```

Comment:
Missing cleanup.

Reply:
Oh sorry I see it is below, did you verify it actually worked? I remember baseapps had issues.

Reply:
> Missing cleanup.

You missed it:
https://github.com/scx/flathub/blob/com.notepadqq.Notepadqq/com.notepadqq.Notepadqq.yaml#L41-L43

Moreover, there are not many things to remove. There is no *.a, *.la or *.pc files, nor man pages or something like that.
```
$ find /app/ \( -name '*.a' -o -name '*.la' \)
$ 
```

If you think that something is really superfluous, then tell us what it is.